contrib/dockerfile_build.bzl

```diff
@@ -15,9 +15,10 @@
 """Repository rule for building a Docker image from a Dockerfile and saving it
    into a tar file.
 
-This rule uses the docker tool installed in the system to build the image and
-then save it as a tar file. The saved tar file is available for other rules to
-use (e.g. container_image, container_load, container_test)
+This rule uses the docker tool installed in the system to build the image or run
```

Comment:
This rule uses either the docker tool installed in the system or the Kaniko executor container to build the image and then save it as a tar file.

Reply:
Done.